created 7. Burning assets chapter (#348)
hyperledger/iroha-2-docs: src/guide/kotlin-java.md

:::

## 7. Burning assets

> Burning assets is quite similar to minting them. Add a couple of lines 
> to the `main()` method.

Reviewer: ```suggestion
Burning assets is quite similar to minting them. First. let's add the following lines 
to the `main()` method:
```

```Kotlin
    sendTransaction.burnAssets(joeAsset, 10, joe.asAccountId(), joeKeyPair)
        .also { println("${joe.asAccountId()} WAS BURN") }

    query.getAccountAmount(joe, joeAsset).also { println("$joeAsset BALANCE: $it AFTER ASSETS BURNING") }
```

> Implement a wrapper over the `burnAssets()` method in the `sendTransaction` class.

Reviewer: ```suggestion
Then implement a wrapper over the `burnAssets()` method in the `sendTransaction` class:
```
